arctic_training/trainer/trainer.py

```diff
@@ -313,6 +317,8 @@ def epoch(self) -> None:
 
             if self.train_batch_idx % self.config.train_log_iter_interval == 0:
                 self.metrics.print_summary()
+                if self.wandb_experiment is not None:
+                    self.wandb_experiment.log(self.metrics.summary_dict)
```

Comment:
we want to tell wandb which step we are logging explicitly:
```suggestion
                    self.wandb_experiment.log(self.metrics.summary_dict, step=self.model.global_steps)
```

Reply:
this allows us to e.g. skip iter 1 as it massively impacts the plot on many metrics and makes the plot more difficult to read.

that way we could go back to logging stats for iter 1 for the normal logging, but skip it for wandb.